include/libelfmaster.h

```diff
@@ -724,4 +754,20 @@ typedef enum typewidth {
 
 bool elf_read_address(elfobj_t *, uint64_t, uint64_t *, typewidth_t);
 bool elf_read_offset(elfobj_t *, uint64_t, uint64_t *, typewidth_t);
+
+
+bool elf_has_header(const char *, bool *, elf_error_t *); 
```

Comment:
When you get a chance put comments describing each function and its arguments. I need to do this too on some of mine. This is where we document our API for other developers who want to work on it.